server-worker/src/main/java/com/aliyun/emr/rss/service/deploy/worker/Writer.java

```diff
@@ -47,8 +48,8 @@
  * Note: Once FlushNotifier.exception is set, the whole file is not available.
  *       That's fine some of the internal state(e.g. bytesFlushed) may be inaccurate.
  */
-public final class FileWriter extends DeviceObserver {
-  private static final Logger logger = LoggerFactory.getLogger(FileWriter.class);
+public final class Writer implements DeviceObserver {
```

Comment:
I think FileWriter is fine, hdfs files and oss files are files, anyway.

Reply:
ok